sal_interfaces/Guider/Guider_Commands.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>Guider</Subsystem>
+    <EFDB_Topic>Guider_command_startGuiding</EFDB_Topic>
+    <Description>Start guiding using the input information. The command returns as soon as guiding starts, before the first interation is completed. If it fails to find a guide star in the provided guider region, the CSC will go to fault state.</Description>
+    <item>
+      <EFDB_Name>roiX</EFDB_Name>
+      <Description>Vector with 8 elements specifying the center of the guider ROI window in the x-axis (in pixels) for each guider sensor. If an element of the array is zero, that guider will be ignored.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>8</Count>
+    </item>
+    <item>
+      <EFDB_Name>roiY</EFDB_Name>
+      <Description>Vector with 8 elements specifying the center of the guider ROI window in the x-axis (in pixels) for each guider sensor. If an element of the array is zero, that guider will be ignored.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>8</Count>
+    </item>
+    <item>
+      <EFDB_Name>roiWidth</EFDB_Name>
+      <Description>Vector with 8 elements specifying the width (window size in the x-axis) of the guider ROI for each guider sensor (in pixels). For each valid entry in roiX/roiY (roiX,roiY &gt; 0) width must respect the ROI constrains (see requirement CA-TS-GDR-ICD-0006 in LSE-66).</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>8</Count>
+    </item>
+    <item>
+      <EFDB_Name>roiHeight</EFDB_Name>
+      <Description>Vector with 8 elements specifying the height (window size in the y-axis) of the guider ROI for each guider sensor (in pixels). For each valid entry in roiX/roiY (roiX,roiY &gt; 0) height must respect the ROI constrains (see requirement CA-TS-GDR-ICD-0006 in LSE-66).</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>8</Count>
+    </item>
+    <item>
+      <EFDB_Name>expTime</EFDB_Name>
+      <Description>Exposure time in milliseconds. All guider sensors will use the same exposure time (CA-TS-GDR-ICD-0034 LSE-66). Limits must respect detector integration time constrains (see requirement CA-TS-GDR-ICD-0034 in LSE-66).</Description>
```

Comment:
The Astropy preffered abbervation is ms:
```
In [21]: u.Unit('millisecond')                                                                           
Out[21]: Unit("ms")
```
also consider if ms are rellay needed. Will s not suffice?

Reply:
Nice, I'll update to `ms`... I'm torn about using `seconds`... although I agree with you it would suffice, the ICD specifies exposure time in milliseconds.